TinyConsole/TinyConsole.swift

```diff
@@ -88,6 +88,13 @@ open class TinyConsole {
     public static func addMarker() {
         TinyConsole.print("-----------", color: UIColor.red)
     }
+    
+    // MARK: - Actions
+    /**
+     3 Finger Tap action delegate
+     */
+    public weak static var threeTapDelegate: TinyConsoleThreeTapDelegate?
```

Comment:
I thought it might be simpler and more consistent with other APIs by setting `TinyConsole.threeTapDelegate` rather than `TinyConsole.shared.threeTapDelegate`. Since I found you are deprecating old functions and replace them with static ones. Would it be better that `TinyConsole` is never initiated and APIs are always static `func` / `var` ?